app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :begin_date, :end_date
+
+  belongs_to :user
```

Comment:
do you need the user? the endpoints you have so far are for a specific user all the time